docs/index.html

```diff
@@ -43,8 +41,11 @@
   <!--[if lt IE 7]>
   <p class="chromeframe">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> or <a href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p>
   <![endif]-->
-  <noscript>
-    <p align="center">You have JavaScript turned off in your browser. To use this website, JavaScript needs to be enabled. You can come back when you turn it on though :)</p>
+  <noscript style="display: flex; color:red;">
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-exclamation-triangle-fill" viewBox="0 0 16 16">
```

Comment:
Hi @theblobscp,

What does the `class` here do? Do we have any declaration for it in the CSS file?

Also, watch out for the `width` and `height` attribute since that's a bit small. If it's intentional, that's alright

Reply:
I used the icon from [bootstrap icons](icons.getbootstrap.com), so that's where the class came from. I'm gonna removed it now